docs/storage/noql.adoc

```diff
@@ -283,6 +283,49 @@ fulltext('data.*', '"gone fishing today"', "AND")
 fulltext('data.*', '"gone fishing today"~2', "AND")
 ----
 
+[#stemmed]
+==== stemmed()
+
+The stemmed function is similar to <<#fulltext,fulltext()>> except that it searches language optimized tokens instead of a source text. E.g. source text `The monkey loved bananas` will be transformed to `the`, `monkey`, `love`, `banana` tokens and they will be used for search.
+
+NOTE: Stemming is language-dependant so content needs a language to be set for stemming to work !
```

Comment:
Here the word content is used. What about when you are trying to do this on the node level? I can't get it to work.

Reply:
It does not work on the node level currently because of the following reasons:
- node is not guaranteed to has language set
- but the most importantly, `LanguageConfigProcessor`, which extracts language to be used by stemming, is only applied to content
- and finally, _LanguageConfigProcessor_ only adds language to `AllTextIndexConfig`, which is later used by `AllTextTypeFactory` to create stemmed `allText` property. But there is also `StemmedTypeFactory` to create stemmed versions for other fields, which seems to be never invoked, because language is not added to `IndexConfigDocument` used by `StemmedTypeFactory`.

So although it does not work now, I do not see reasons why it can't potentially be done for nodes as well.